apps/namadillo/src/atoms/fees/services.ts

```diff
@@ -1,24 +1,53 @@
-import { DefaultApi } from "@anomaorg/namada-indexer-client";
+import {
+  DefaultApi,
+  GasLimitTableInnerTxKindEnum as GasLimitTableIndexer,
+} from "@anomaorg/namada-indexer-client";
+import { assertNever, mapUndefined } from "@namada/utils";
 import BigNumber from "bignumber.js";
 import invariant from "invariant";
 import { GasTable } from "types";
-import { txKindFromIndexer } from "./atoms";
+import { TxKind, txKinds } from "types/txKind";
+
+const txKindToIndexer = (txKind: TxKind): GasLimitTableIndexer => {
```

Comment:
Nitpick: maybe this should be in `functions.ts` instead of services?

Reply:
Fixed!